safe/gis/generic_expressions.py

```diff
@@ -287,14 +295,21 @@ def beautify_time(inasafe_time, feature, parent):
     """
     _ = feature, parent  # NOQA
     try:
-        datetime_object = datetime.datetime.strptime(
-            inasafe_time, '%Y-%m-%dT%H:%M:%S.%f')
-    except ValueError:
+        gmt_offset_str = inasafe_time[-6:-4]
```

Comment:
It seems duplicate code with the one in beautify_date. Can you make it as a function?

Reply:
+1